llvm/lib/Analysis/ValueTracking.cpp

```diff
@@ -1121,6 +1121,40 @@ static void computeKnownBitsFromOperator(const Operator *I,
       break;
     }
 
+    Value *V;
+    // Handle bitcast from floating point to integer.
+    if (match(const_cast<Operator *>(I), m_ElementWiseBitCast(m_Value(V))) &&
```

Comment:
Shouldn't need const cast if V is const?

Reply:
Done.